File: Api/models/Pelicula.js
'use strict';

const { Sequelize } = require('sequelize');
const sequelize = new Sequelize({
    dialect: 'sqlite',
    storage: './storage/bdpeli.sqlite'
});

const Pelicula = sequelize.define('Pelicula', {
    id: {
        primaryKey: true,
        type: Sequelize.BIGINT,
        autoIncrement: true
    },
    titulo: {
        type: Sequelize.STRING

    },
    genero: {
        type: Sequelize.STRING
    },
    director: {
        type: Sequelize.STRING
    },
    anio: {
        type: Sequelize.STRING
    },

});

Pelicula.sync();

module.exports = Pelicula;

File: PelisItic92Yen/PelisItic92Yen/Data/Pelicula.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PelisItic92Yen.Data
{
    public class Pelicula
    {
        public long Id { get; set; }

        public string Titulo { get; set; }

        public string Genero { get; set; }

        public string Director { get; set; }

        public string Anio { get; set; }
    }
}
